sui/src/unit_tests/rpc_server_tests.rs

```diff
@@ -185,6 +190,94 @@ async fn test_move_call() -> Result<(), anyhow::Error> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_get_object_info() -> Result<(), anyhow::Error> {
+    let test_network = setup_test_network().await?;
+    let http_client = test_network.http_client;
+    let address = test_network.accounts.first().unwrap();
+    http_client.sync_account_state(*address).await?;
+    let result: ObjectResponse = http_client.get_owned_objects(*address).await?;
+    let result = result
+        .objects
+        .into_iter()
```

Comment:
Nit, this could probably be an `iter` here and below.

Reply:
oh... `to_object_ref` ' s signature is `pub fn to_object_ref(self) -> Result<ObjectRef, anyhow::Error>`, so this need to be `into_iter`

Reply:
fair enough!